translations/ui-export-manager/en.json

```diff
@@ -47,7 +47,7 @@
   "exportJob.details.action.download": "Download",
   "exportJob.details.action.download.error": "Your file download was failed.",
   "exportJob.details.action.rerun": "Rerun",
-  "exportJob.details.action.rerun.success": "Export job <b>{name}</b> was successfully reruned.",
+  "exportJob.details.action.rerun.success": "Export job <b>{name}</b> was successfully rerun.",
```

Comment:
When the value of the translation is changed it is necessary to change the key of translation or move it to another line to trigger the translation bot. The change won't apply in the current implementation.

Reply:
Yeah, but we can cordially ask Peter Murray to update them so as not to change keys)

Reply:
I don't think those statements are correct, @Dmytro-Melnyshyn. This fix looks good to me and is both necessary and sufficient.